packages/vue-flicking/demo/features/Align.vue

```diff
@@ -11,25 +11,25 @@
     <h3>hanger: 50%, anchor: 50%</h3>
     <div class="hanger flicking0"></div>
     <flicking class="flicking flicking0">
-      <div class="panel"><div class="anchor"></div></div>
-      <div class="panel"><div class="anchor"></div></div>
-      <div class="panel"><div class="anchor"></div></div>
-      <div class="panel"><div class="anchor"></div></div>
-      <div class="panel"><div class="anchor"></div></div>
-      <div class="panel"><div class="anchor"></div></div>
+      <div class="panel0"><div class="anchor"></div></div>
+      <div class="panel1"><div class="anchor"></div></div>
+      <div class="panel2"><div class="anchor"></div></div>
+      <div class="panel3"><div class="anchor"></div></div>
+      <div class="panel4"><div class="anchor"></div></div>
+      <div class="panel0"><div class="anchor"></div></div>
```

Comment:
@WoodNeck Is it better to have `panel5`?

Reply:
@WoodNeck Or How about removing last `panel0`?

Reply:
Well, I remember that all other demos use 5 panels at max, so that's quite exceptional.
I think removing the last panel0 is good.